scanTopic: /scan
queueSize: 50
scanPublisherTopic: /recreatedScan
velPublisherTopic: /cmd_vel
kP: 1.0
collisionThreshold: 1.0
visuMarker: /visuMarker
huskyMovePublisherTopic: /huskyMove
distanceToWall: 1.1
acCancelTopic: /acCancelTopic